### DISCUSSION TITLE
Inverted Image

### DISCUSSION THREAD

**Discussion by Ranbud** [arkypita/LaserGRBL#1949] [Q&A]
When I load a image, I start to burn it, but it burns the image upside down. I can invert the image on the screen to get it to burn right, but is there a setting that I am missing that causes this?

**Comment by Tribo-2dot0** [comment]
Adjust the direction of the axes according to the table, modifying the value of $3.
![Captura de Tela (181)](https://user-images.githubusercontent.com/83982525/199871517-4be710dd-86cf-49d5-b52f-626847647bfa.png)
